src/index.html

```diff
@@ -6,9 +6,109 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="head">
+      <a
+        href="#home"
+        class="logo-moyo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo-Logo"
+        >
+      </a>
+      <nav
+        class="nav"
+      >
+        <ul class="nav__list">
+          <li
+            class="nav__item"
+          >
```

Comment:
```suggestion
          <li class="nav__item">
```
If you have short tags you may place them in one line